packages/forc-bin/package.json

```diff
@@ -22,6 +22,7 @@
   "scripts": {
     "preinstall": "pnpm build",
     "install": "./dist/install.js",
+    "setup": "pnpm build",
```

Comment:
here we have two scripts equal, and we are calling both on. We should have only one...
```
"prebuild": {
      "dependsOn": ["^prebuild", "setup"]
    },
```